daemon/graphdriver/devmapper/driver.go

```diff
@@ -151,7 +151,7 @@ func (d *Driver) Remove(id string) error {
 
 	// This assumes the device has been properly Get/Put:ed and thus is unmounted
 	if err := d.DeviceSet.DeleteDevice(id, false); err != nil {
-		return err
+		return fmt.Errorf("failed to remove device %v:%v", id, err)
```

Comment:
We try to use it more by default. The `errors.Wrap` keeps the original error intact, which in cases can help to retrieve the underlying error, i.e., the _original_ error can be obtained through `errors.Cause(wrappedError)`. Maybe not used in this case, just adding more context to the error